### ISSUE TITLE
Warmup and learning rate

### ISSUE THREAD

**Issue by swapnil-saha** [open-mmlab/mmcv#512]
How does warmup affect the learning rate at each iteration?

**Comment by xvjiarui** [comment]
For the linear warmup schedule, the learning rate will be increased linearly as iteration goes.

**Comment by swapnil-saha** [comment]
thank you for your reply

**Comment by swapnil-saha** [comment]
What does step do in warm up iteration ?

**Comment by xvjiarui** [comment]
Hi @swapnil-saha 
There is no step argument for warm up. For `StepLrUpdater`, `step` indicates at which training step(epoch) lr will be decreased.
